src/components/LanguageSwitcher.tsx

```
@@ -0,0 +1,163 @@
+
+    const currentIndex = itemRefs.current.findIndex((ref) => ref === document.activeElement)
+
+    switch (event.key) {
+      case "Escape":
+        event.preventDefault()
+        setIsOpen(false)
+        break
+      case "ArrowDown":
+        event.preventDefault()
+        const nextIndex = (currentIndex + 1) % languages.length
+        itemRefs.current[nextIndex]?.focus()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)